infrastructure/authors/authors.go

```diff
@@ -8,18 +8,24 @@ import (
 )
 
 var (
-	ErrNotFound   = errors.New("author could not be found")
+	// ErrNotFound is returned when a record cannot be found
+	ErrNotFound = errors.New("author could not be found")
+	// ErrNotCreated is returned when a record cannot be created
 	ErrNotCreated = errors.New("author could not be created")
 )
 
+// New creates a new Repository instance for authors
 func New(db *gorm.DB) *Repository {
 	return &Repository{db: db}
 }
 
+// Repository is used for querying and creating authors
 type Repository struct {
 	db *gorm.DB
 }
 
+// GetByID returns an author from the database given an ID
```

Comment:
To be pedantic, they both need periods.

Reply:
I'm sorry, did you forget this is golang? We don't end our lines with punctuation. Ever. Period.

Reply:
> Doc comments work best as complete sentences, which allow a wide variety of automated presentations. The first sentence should be a one-sentence summary that starts with the name being declared.

😇